ivy/functional/frontends/jax/numpy/statistical.py

```diff
@@ -512,3 +514,29 @@ def quantile(
     return ivy.quantile(
         a, q, axis=axis, keepdims=keepdims, interpolation=method, out=out
     )
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes(
+    {"0.4.14 and below": ("complex64", "complex128", "bfloat16", "bool", "float16")},
+    "jax",
+)
+def nanquantile(
+    a,
+    q,
+    axis=None,
+    out=None,
+    overwrite_input=False,
+    method="linear",
+    keepdims=False,
+    interpolation=None,
+):
+    if interpolation:
+        warnings.warn(
++            "interpolation is deprecated name for the method keyword argument since"
++            " version 1.22.0 in JAX"
++        )
++       method = interpolation  
+    return ivy.nanquantile(
+        a, q, axis=axis, keepdims=keepdims, interpolation=method, out=out
+    )
```

Comment:
this function was removed from the functional API just a while ago, so either implement it again or use a compositional implementation, perhaps a combination of `where` and `quantile`

Reply:
Hi @MahmoudAshraf97 
Thank you very much for your helpful review.
I thought I could use the nanquantile function from the ivy/experimental
https://github.com/geeythree/ivy/blob/9b8bb5c78b14070e7ee168de3a8c533fae3f8655/ivy/functional/ivy/experimental/statistical.py#L498
If that can't work, please let me know.

Reply:
as I mentioned, the function was removed, you are right, you could've used it if it wasn't removed so this is not your fault

Reply:
Hi @MahmoudAshraf97 
Thank you very much for clarifying that. 

I have one last question, Is there an IVY function that can filter Nans or should I just use Numpy?
If I used NumPy, it would probably look something like this: ```a = a[~numpy.isnan(a)]```

Reply:
that sounds plausible to me, try it and see if the tests pass